tests/sourcerer/access_credentials/test_services.py

```
@@ -0,0 +1,465 @@
+import json
+import unittest
+from unittest.mock import MagicMock, patch
+
+import boto3
```

Ruff (0.8.2)

`boto3` imported but unused

Remove unused import: `boto3`

(F401)